src/widgets/index/index.ts

```diff
@@ -21,6 +21,8 @@ import {
   createDocumentationMessageGenerator,
   resolveSearchParameters,
   mergeSearchParameters,
+  warning,
+  capitalize,
```

Comment:
just wondering here, what have we used capitalize for before?

Reply:
We use it to compute the CurrentRefinements label:

https://github.com/algolia/instantsearch.js/blob/9b0d79f0915bcb0fa9662097530b9448a1538997/src/components/CurrentRefinements/CurrentRefinements.tsx#L44